doc/classes/AcceptDialog.xml

```diff
@@ -67,6 +67,8 @@
 		<member name="dialog_text" type="String" setter="set_text" getter="get_text" default="&quot;&quot;">
 			The text displayed by the dialog.
 		</member>
+		<member name="window_title" type="String" setter="set_title" getter="get_title" default="&quot;Alert!&quot;" override="true">
```

Comment:
We probably want to avoid having to copy paste or rewrite the base class' property description for overridden properties, so maybe they should be in a separate category that doesn't take a description? E.g. `<overridden>`.

Reply:
Property descriptions are ignored for `override="true"` members. `<overridden_members>` sounds good as well, but I wasn't sure if I want to go this way...

Reply:
Most contributors won't be aware of this, or even notice the `override` parameter though, so they'll end up writing docs that will be dropped by doctool.

Reply:
Maybe I can write override members with a self-closing tag, as in `<member .... />`?

Reply:
That could work :)

Reply:
Did it that way -- less work ^^.